libutils/string_lib.c

```diff
@@ -372,13 +372,140 @@ bool EmptyString(const char *s)
 
 /*********************************************************************/
 
-long StringToLong(const char *str)
+/**
+ * @brief Converts a string of numerals in base 10 to a long integer.
+ *
+ * Result is stored in *number, return value should be chacked.
+ * On error *number is unmodified and -1 is returned.
+ * Leading spaces in input string are skipped.
+ * String numeral must be terminated by NULL byte or space (isspace).
+ *
+ * @see StringToLongExitOnError()
+ * @see StringToLongDefaultOnError()
+ * @param[in] str String with numerals to convert, cannot be NULL
+ * @param[out] number Where to store result on success, cannot be NULL
+ * @return 0 on success, error code otherwise (see source code)
+ */
+int StringToLong(const char *str, long *number)
```

Comment:
I would suggest to rename `number` to `value_out` in both the C and header file so that it's obvious this is an "out" parameter.

Reply:
Thanks, done.